recipe_scrapers/projectgezond.py

```diff
@@ -11,82 +10,32 @@ def host(cls):
-
-    def total_time(self):
-        time_element = self.soup.find("em", string="Bereidingstijd:").parent
-        return "".join(
-            [
-                element.text
-                for element in time_element.children
-                if element.text != "Bereidingstijd:"
-            ]
-        ).strip()
-
-    def yields(self):
-        # Match everything in the h2 with 'Dit heb je nodig'
-        # The text inside the parentheses contains the yield for the ingredients that are listed
-        return re.search(
-            r"Dit heb je nodig \(([^)]+)",
-            self.soup.find(
-                "h2", string=lambda x: x.startswith("Dit heb je nodig")
-            ).text,
-        ).group(1)
-
-    def ingredients(self):
-        ingredients_table = self.soup.find(
-            "h2", string=lambda x: x.startswith("Dit heb je nodig")
-        ).next_sibling.next_sibling
-        ingredients = [
-            ingredient.text
-            for ingredient in ingredients_table
-            if ingredient.text.strip()
-        ]
-        return ingredients
-
-    def instructions(self):
-        instructions_table = self.soup.find(
-            "h2", string=lambda x: x.startswith("Zo maak je het")
-        ).next_sibling.next_sibling.next_sibling.next_sibling
-        instructions = [
-            instruction.text
-            for instruction in instructions_table
-            if instruction.text.strip()
-        ]
-        return "\n".join(instructions).strip()
-
     def ratings(self):
-        # Ratings do not exist on this site
-        return None
+        raise FieldNotProvidedByWebsiteException(return_value=None)
 
     def cuisine(self):
-        # Not listed on site
-        return None
-
-    def description(self):
-        # Get the recipe's content start. The recipe will start with the description until
-        # we reach the instructions.
-        content_start = self.soup.find("div", {"class", "entry-content"})
+        raise FieldNotProvidedByWebsiteException(return_value=None)
 
-        description = ""
-        for content_element in content_start.children:
-            # If we reach this, the ingredients are listed and the description is complete
-            if content_element.text.startswith("Dit heb je nodig"):
-                break
-
-            description += content_element.text
+    def category(self):
+        category_script = self.soup.find(
+            "script", string=lambda text: text and "var dataLayer_content =" in text
+        )
+        if category_script:
+            category_text = category_script.string
+            start = category_text.find('"pageCategory":[')
+            if start != -1:
+                start += len('"pageCategory":[')
+                end = category_text.find("]", start)
+                if end != -1:
+                    categories = category_text[start:end].strip('"').split('","')
+                    return categories if categories else None
+        return None
```

Comment:
Nitpick: I'd suggest inverting the logic here, to reduce the indentation level:

```suggestion
        if not category_script:
            return None

        category_text = category_script.string
        start = category_text.find('"pageCategory":[')
        if start != -1:
            start += len('"pageCategory":[')
            end = category_text.find("]", start)
            if end != -1:
                categories = category_text[start:end].strip('"').split('","')
                return categories if categories else None
```